Sources/Datadog/Logger.swift

```diff
@@ -15,6 +15,30 @@ public enum LogLevel: Int, Codable {
     case warn
     case error
     case critical
+
+    // MARK: - `LogLevel` <> `Log.Status` conversion
```

Comment:
Yes, they are the same and we could remove `Log.Status` in favour of `LogLevel`. This would require `switch / case` in `LogEncoder` to implement encoding of each level as `String`. With `Log.Status: String` this is now automated and this mapping here keeps it consistent.

Reply:
Also, because those are two separate types now, we can adjust one (e.g. make it raw `String` representable + `CaseIterable`) without touching the other (defined in `public` API). If they would become one, we'd have `public`, `CaseIterable`, raw `String` representable, `Encodable` enum in public API. Even today's raw `Int` representable + `Codable` is useless for the public API of `LogLevel` (it shouldn't be there, but as it's public now there's no easy way back).